### DISCUSSION TITLE
When is the target date for the first 1.6.0(.Alpha/Beta) release?

### DISCUSSION THREAD

**Discussion by ge0ffrey** [jreleaser/jreleaser#1311] [Q&A] [answered]
I look forward to the JReleaser 1.6.0 release because the empty jars deploy to Maven Central issue is a blocker for me and it is fixed for 1.6.0.
I tried building JReleaser (including the maven-jreleaser-plugin) from source without much success so far. Using a from source build  would be hard in GitHub Actions, but I could release manually for now.

So, I am wondering... when's the ETA for a 1.6.0 release? Doesn't matter if it's a alpha, beta or final release.

**Comment by aalmiray** [comment, marked as answer]
According to the milestone page it's set for April 29th https://github.com/jreleaser/jreleaser/milestone/25

We post releases every 2 months, at the end of the even month. https://github.com/jreleaser/jreleaser/discussions/873

**Reply by aalmiray** [reply]
Alternatively, you could switch to using the CLI instead of the Maven plugin. early-access releases of the CLI are posted on every push to the default branch.

**Reply by aalmiray** [reply]
Moreover, the maven plugin can consume the same external configuration file as the CLI. This means you can still rely on the maven plugin when using stable releases, and the CLI for early-access, while having a single configuration file.

**Reply by ge0ffrey** [reply]
Thanks for the info, Andres. Much appreciated.

Long term, I am avoid the CLI so the jreleaser version is managed in the pom.xml. That way, I only need to worry that all machines (others, mine and GitHub Actions) have Maven+java installed and use these same Maven+java version - not if they have jreleaser/jbake/asciidoctor/... installed and the correct version of each installed.

I'll try to workaround it with the early access CLI build as you propose. Thank you!
 https://github.com/jreleaser/jreleaser/wiki/Release-early-access
